CONTRIBUTING.md

```diff
@@ -2,9 +2,9 @@
 Thanks for considering contributing to our repo! We appreciate all contributions to the player, large or small. Please take the time to read through this document before making a contribution to ensure a fun and effective process for everyone involved.
 
 ## General Guidelines
-Are you having trouble getting started with JW Player, configuration, or customization? If so, please check the [support docs](https://support.jwplayer.com/) before submitting an issue here. 
+Are you having trouble getting started with JW Player, configuration, or customization? If so, please check the [support docs](https://support.jwplayer.com/) before submitting an issue here.
```

Comment:
We should change this to point to the developer docs https://developer.jwplayer.com/jw-player/docs/developer-guide/

Reply:
Updating!